mappings/net/minecraft/client/render/RenderLayer.mapping

```diff
@@ -68,7 +114,8 @@ CLASS net/minecraft/class_1921 net/minecraft/client/render/RenderLayer
 	FIELD field_20970 LIGHTNING Lnet/minecraft/class_1921;
 	FIELD field_20972 vertexFormat Lnet/minecraft/class_293;
 	FIELD field_20973 drawMode I
-	FIELD field_21402 sortQuads Z
+	FIELD field_20975 hasCrumbling Z
+	FIELD field_21402 translucent Z
```

Comment:
Having to sort quads is a consequence of translucence. I think it's better to be more general and say `translucent` rather than mention one specific effect `sortQuads` (even though it's the current effect, at least for now).